### PR TITLE
call out support for AlloyDB via dbt-postgres

### PR DESCRIPTION
**Pull request by dataders**
## Description & motivation

dbt Core and dbt Cloud both work using AlloyDB, this should be documented here.

[Link to preview of entire docs site](https://deploy-preview-1926--docs-getdbt-com.netlify.app/)

## Changes
This PR adds:
### 1. An entry for AlloyDB to our list of vendor-supported adapters on [Available Adapters](https://deploy-preview-1926--docs-getdbt-com.netlify.app/docs/available-adapters#vendor-supported) 
image
    
### 2. a [profile page for AlloyDB](https://deploy-preview-1926--docs-getdbt-com.netlify.app/reference/warehouse-profiles/alloydb-profile) that redirects users to existing docs for dbt-postgres
image
    
### 3. references to AlloyDB on the [dbt Cloud doc: Connecting to your Database](https://deploy-preview-1926--docs-getdbt-com.netlify.app/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database#connecting-to-postgres-redshift-and-alloydb)
image

## To-do before merge
Include any notes about things that need to happen before this PR is merged, e.g.:
- [ ] Ensure https://github.com/dbt-labs/www.getdbt.com/pull/1355/ (repo private to dbt Labs)is merged, which adds AlloyDB to the [dbt Cloud integrations page](https://www.getdbt.com/product/integrations/) (screenshot below)
    
    image

## Checklist
If you added new pages (delete if not applicable):
- [x] The page has been added to `website/sidebars.js`
- [x] The new page has a unique filename

### CODE REVIEW

**Review by dbeatty10 — APPROVED** [review]
Nice @dataders :elmofire:

File: website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ The following fields are required when creating a Redshift connection:
 
 ### Connecting via an SSH Tunnel
 
-To connect to a Postgres or Redshift instance via an SSH tunnel, check the "Use SSH Tunnel" option when creating your connection. When configuring the tunnel, you'll need to supply the hostname, username, and port for the bastion server.
+To connect to a Postgres, Redshift, or AlloyDB instance via an SSH tunnel, check the "Use SSH Tunnel" option when creating your connection. When configuring the tunnel, you'll need to supply the hostname, username, and port for the bastion server.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]:
```suggestion
To connect to a Postgres, Redshift, or AlloyDB instance via an SSH tunnel, select the **Use SSH Tunnel** option when creating your connection. When configuring the tunnel, you must supply the hostname, username, and port for the bastion server.
```

Comment (nghi-ly) [reply website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-your-database.md]: the docs team is working on a style guide, which we've started to use (and will publish soon!). for checkboxes, "select" and "choose" are the preferred words to use: https://github.com/dbt-labs/product-docs-team/blob/main/content-style-guide.md#checkboxes

**Review by nghi-ly — COMMENTED** [review]
looks great, @dataders ! left one sugg

**Pull request merged**